src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-list/smb-cluster-list.component.ts

```diff
@@ -28,17 +35,33 @@ export class SmbClusterListComponent extends ListWithDetails implements OnInit {
   permission: Permission;
   tableActions: CdTableAction[];
   context: CdTableFetchDataContext;
-
   smbClusters$: Observable<SMBCluster[]>;
   subject$ = new BehaviorSubject<SMBCluster[]>([]);
+  selection = new CdTableSelection();
+  modalRef: NgbModalRef;
 
   constructor(
     private authStorageService: AuthStorageService,
     public actionLabels: ActionLabelsI18n,
-    private smbService: SmbService
+    private smbService: SmbService,
+    private modalService: ModalCdsService,
+    private taskWrapper: TaskWrapperService
   ) {
     super();
     this.permission = this.authStorageService.getPermissions().smb;
+    this.tableActions = [
+      {
+        permission: 'update',
+        icon: Icons.edit,
+        name: this.actionLabels.EDIT
+      },
```

Comment:
why this ? We must add the feature with its implementation.
```suggestion
```

Reply:
@afreen23 , if i remove this section(edit) "Remove" table action is displayed at top right corner as shown below.
![Screenshot from 2024-12-20 17-45-31](https://github.com/user-attachments/assets/706f9b1e-3647-404d-bc89-b2811410ebea)

Reply:
@dnyanee1997  That should not be a problem, since we only have the remove for now that's what it will display, once we introduce the edit functionality we can add the button as well and have that one first.
But I agree with Afreen, let's not push code unrelated to the PR/commit

Reply:
Agree.